src/phasorpy/phasor.py

```diff
@@ -444,18 +450,87 @@ def phasor_calibrate(
     (array([-1.56, 1.934, 0.3279]), array([5.985, 10.6, 1.986]))
 
     """
-    phi0 = numpy.asarray(phase0)
-    mod0 = numpy.asarray(modulation0)
-    if phi0.shape != mod0.shape:
-        raise ValueError(f'{phi0.shape=} != {mod0.shape=}')
+    re = numpy.asarray(real)
+    im = numpy.asarray(imag)
+    if re.shape != im.shape:
+        raise ValueError(f'{re.shape=} != {im.shape=}')
+    ref_re = numpy.asarray(reference_real)
+    ref_im = numpy.asarray(reference_imag)
+    if ref_re.shape != ref_im.shape:
+        raise ValueError(f'{ref_re.shape=} != {ref_im.shape=}')
+    kwargs_center = {key: value for key, value in kwargs.items() if key in inspect.signature(phasor_center).parameters}
+    measured_re, measured_im = phasor_center(reference_real, reference_imag, **kwargs_center)
+    kwargs_lifetime = {key: value for key, value in kwargs.items() if key in inspect.signature(phasor_from_lifetime).parameters}
+    known_re, known_im = phasor_from_lifetime(frequency, lifetime, fraction, **kwargs_lifetime)
+    phi_shift, mod_ratio = polar_from_reference_phasor(measured_re, measured_im, known_re, known_im)
+    return phasor_transform(re, im, phi_shift, mod_ratio)
+
+def phasor_transform(
+    real: ArrayLike,
+    imag: ArrayLike,
+    phase_shift: ArrayLike = 0.0,
+    modulation_ratio: ArrayLike = 1.0,
+    /,
+) -> tuple[NDArray[Any], NDArray[Any]]:
+    """Return transformed/rotated phasor coordinates.
+
+    This function is used for calibration purposes, but can also be used to
+    transform and/or rotate phasor coordinates. The transform is applied in
+    the polar system and returned as phasor coordinates.
+
+    Parameters
+    ----------
+    real : array_like
+        Real component of phasor coordinates to transform/rotate.
```

Comment:
-> "Real component of phasor coordinates to transform."
Just refer to "transform", or alternatively "rotate and scale".